packages/graphql/src/api-v6/schema-generation/schema-types/TopLevelEntitySchemaTypes.ts

```diff
@@ -324,6 +359,19 @@ export class TopLevelEntitySchemaTypes {
             };
         });
     }
+
+    public get updateInfo(): ObjectTypeComposer {
+        return this.schemaBuilder.getOrCreateObjectType(this.entityTypeNames.createInfo, () => {
```

Comment:
This should be `updateInfo`
```suggestion
        return this.schemaBuilder.getOrCreateObjectType(this.entityTypeNames.updateInfo, () => {
```
But actually, I noticed that is not related to any ConcreteEntity, therefore, can be moved to the static types same for the `createInfo`